libs/react/src/lib/modal/modal.tsx

```diff
@@ -66,14 +66,18 @@ const ModalFooter = ({
       )}
     </div>
   )
-}
+})
 
-export const Modal = ({ type = 'default', isOpen, ...props }: ModalProps) => {
+export const Modal = memo(({ type = 'default', isOpen, size = 'sm', ...props }: ModalProps) => {
   const modalContent = () => {
     switch (type) {
       case 'slideout': {
+        let className: string | undefined = undefined;
+        if (size === "lg") className = 'gds-slide-out--960';
+        if (size === "md") className = 'gds-slide-out--768';
+
         return (
-          <aside role="dialog">
+          <aside role="dialog" className={className} aria-modal="true" aria-labelledby="modal_header" aria-describedby="modal_body">
```

Comment:
added accessibility area props also, same as the ones used in the chlorophyl storybook